src/components/TranslationsInProgress.js

```diff
@@ -1,33 +1,17 @@
 import React, { useState, useEffect } from "react"
-import { Link as GatsbyLink } from "gatsby"
 import styled from "styled-components"
 import axios from "axios"
 import { FakeLinkExternal } from "./SharedStyledComponents"
 import Translation from "./Translation"
+import { CardItem } from "./SharedStyledComponents"
 
 const LangContainer = styled.div`
   margin-bottom: 2rem;
   display: flex;
   flex-wrap: wrap;
 `
-
-const LangItem = styled(GatsbyLink)`
-  text-decoration: none;
-  margin: 1rem 1rem 1rem 0;
-  padding: 1rem;
+const LangItem = styled(CardItem)`
```

Comment:
This renders with an extra "external" arrow glyph. By switching this to use `CardItem`, it now treats this link differently and adds an arrow glyph. This is already accommodated for in the `FakeExternalLink` which puts the arrow next to the appropriate text.

tl;dr - Add `hideArrow` attribute to L42 to hide extra arrow:
`<LangItem to={url} key={lang.code} hideArrow>`